Fix erroneous 'const' modifiers in ppmgr driver.
Older gccs would ignore it, newer treat as errors.

diff -ur original/drivers/amlogic/media/video_processor/ppmgr/ppmgr_drv.c modified/drivers/amlogic/media/video_processor/ppmgr/ppmgr_drv.c
--- original/drivers/amlogic/media/video_processor/ppmgr/ppmgr_drv.c	2018-09-21 07:30:58.000000000 +0300
+++ modified/drivers/amlogic/media/video_processor/ppmgr/ppmgr_drv.c	2018-12-16 13:52:52.149444204 +0300
@@ -828,7 +828,7 @@
 static ssize_t viewmode_read(
 	struct class *cla, struct class_attribute *attr, char *buf)
 {
-	const char *viewmode_str[] const = {
+	const char *viewmode_str[] = {
 		"normal", "full", "4:3", "16:9", "1:1"};
 	return snprintf(buf, 80, "current view mode is %d:%s\n",
 			ppmgr_device.viewmode,
@@ -857,7 +857,7 @@
 static ssize_t doublemode_read(
 	struct class *cla, struct class_attribute *attr, char *buf)
 {
-	const char *doublemode_str[] const = {
+	const char *doublemode_str[] = {
 		"normal", "horizontal double", "vertical double"};
 	unsigned int mode = get_ppmgr_3dmode();
 
@@ -894,7 +894,7 @@
 static ssize_t switchmode_read(
 	struct class *cla, struct class_attribute *attr, char *buf)
 {
-	const char *switchmode_str[] const = {"disable", "enable"};
+	const char *switchmode_str[] = {"disable", "enable"};
 	unsigned int mode = get_ppmgr_3dmode();
 	unsigned int flag = (mode & PPMGR_3D_PROCESS_SWITCH_FLAG) ? 1 : 0;
 
@@ -930,7 +930,7 @@
 static ssize_t direction_3d_read(
 	struct class *cla, struct class_attribute *attr, char *buf)
 {
-	const char *direction_str[] const = {
+	const char *direction_str[] = {
 		"0 degree", "90 degree", "180 degree", "270 degree"};
 /*unsigned mode = get_ppmgr_3dmode();*/
 /* mode = ((mode & PPMGR_3D_PROCESS_3D_ROTATE_DIRECTION_MASK)
@@ -973,7 +973,7 @@
 static ssize_t scale_down_read(
 	struct class *cla, struct class_attribute *attr, char *buf)
 {
-	const char *value_str[] const = {"noraml", "div 2", "div 3", "div 4"};
+	const char *value_str[] = {"noraml", "div 2", "div 3", "div 4"};
 	unsigned int mode = ppmgr_device.scale_down;
 
 	return snprintf(buf, 80, "current scale down value is %d:%s\n",
